docs/pact-reference.md

```diff
@@ -1104,33 +1105,31 @@ Tables and objects can only take a schema type literal.
 Special forms {#special-forms}
 ---
 
-### Docs and metadata
+### Docs and Metadata
 Many special forms like [defun](#defun) accept optional documentation strings,
 in the following form:
 
 ```lisp
-(defun foo (bar)
-  "Do the thing with BAR"
-  ...)
+(defun average (a b)
+  "take the average of a and b"
+  (/ (+ a b) 2))
 ```
 
-However, in this position an optional _metadata section_ can specify docs and
-metadata, where metadata can be tagged with any key desired.
-The following code provides a docstring of "does the thing with BAR" and specifies metadata of type
-`property` and `example`:
+However, in this position, extra metadata fields can also be specified.
```

Comment:
This should specifically mention `@doc` and `@model`, ie 
> Alternately, users can specify metadata using a special `@`-prefix syntax. Supported metadata fields are `@doc` to provide a documentation string, and `@model` that can be used by Pact tooling ...

Reply:
Both `@model` and `@doc` are mentioned immediately afterwards, so would you still like this change?

Reply:
Yes, as I was trying to indicate above, my edit ends on line 1120 with "Pact tooling ..."